client/components/domains/domain-registration-suggestion/index.jsx

```diff
@@ -34,11 +34,18 @@ const DomainRegistrationSuggestion = React.createClass( {
 
 	componentDidMount() {
 		if ( this.props.railcarId && isNumber( this.props.uiPosition ) ) {
+			let resultSuffix = '';
+			if ( this.props.suggestion.isRecommended ) {
+				resultSuffix = 'recommended';
+			} else if ( this.props.suggestion.isBestAlternative ) {
+				resultSuffix = 'best-alternative';
+			}
+
 			tracks.recordEvent( 'calypso_traintracks_render', {
 				railcar: this.props.railcarId,
 				ui_position: this.props.uiPosition,
 				fetch_algo: this.props.fetchAlgo,
-				rec_result: this.props.suggestion.domain_name,
+				rec_result: `${ this.props.suggestion.domain_name }#${ resultSuffix }`,
```

Comment:
Oops! I moved the `#` into the `resultSuffix` variable.

This was the format that Sirin suggested using to add this information in the comment above. We don't have a way of adding arbitrary fields for a `_traintracks_render` event as far as I know.